examples/hello_lib/BUILD.bazel

```diff
@@ -28,6 +28,13 @@ rust_static_library(
     ],
 )
 
+# A helper library used only in doc tests.
```

Comment:
There's actually a unit test section for rustdoc, can you add coverage there instead?
https://github.com/bazelbuild/rules_rust/tree/0.2.1/test/unit/rustdoc

Reply:
Done.